pandas/tests/arithmetic/conftest.py

```diff
@@ -4,13 +4,49 @@
 import numpy as np
 import pandas as pd
 
+from pandas.compat import long
+
 
 @pytest.fixture(params=[1, np.array(1, dtype=np.int64)])
 def one(request):
     # zero-dim integer array behaves like an integer
     return request.param
 
 
+zeros = [box_cls([0] * 5, dtype=dtype)
+         for box_cls in [pd.Index, np.array]
+         for dtype in [np.int64, np.uint64, np.float64]]
+zeros.extend([np.array(0, dtype=dtype)
+              for dtype in [np.int64, np.uint64, np.float64]])
+zeros.extend([0, 0.0, long(0)])
+
+
+@pytest.fixture(params=zeros)
+def zero(request):
+    # For testing division by (or of) zero for Index with length 5, this
+    # gives several scalar-zeros and length-5 vector-zeros
+    return request.param
+
+
+@pytest.fixture(params=[pd.Float64Index(np.arange(5, dtype='float64')),
```

Comment:
call this something else, idx is too specific a name (and it is used heavily elsewhere)

Reply:
Will do in next pass (hopefully tomorrow)